lint-staged.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  '*.+(js|jsx|ts|tsx)': ['yarn lint'],
+  // '*.+(js|jsx|ts|tsx)': ['eslint', 'yarn test:precommit'],
+  '*.+(js|jsx|json|yml|yaml|css|ts|tsx)': ['prettier --write'],
```

Comment:
prettier should be included in eslint config in `yarn lint`, no need to repeat on those files, only on `json|yml|yaml|css`, no?

Reply:
for yml its helpful because it check its format. that is why i kept it separate